frontend/src/containers/map/content/map/analysis/index.tsx

```diff
@@ -14,10 +14,11 @@ const fetchAnalysis = async () => {
   return new Promise((resolve) => {
     setTimeout(() => {
       resolve({
-        data: {
-          locations_area: [{ FRA: 23856, USA: 502367 }],
-          total_area: 736000,
-        },
+        locations_area: [
+          { code: 'FRA', protected_area: 2385406 },
+          { code: 'USA', protected_area: 5000367 },
+        ],
+        total_area: 73600000,
```

Comment:
Have you talked to Alicia about this change?

Reply:
Unfortunately by the time I got to this Alicia was off. The endpoint wasn't ready (nor mocked) at this time so I took a couple liberties in order to keep the implementation going; some talk about this [here](https://vizzuality.slack.com/archives/C056QHB3TML/p1701780715046399). 

@agnessa now implemented a quick mocked response in the endpoint matching this; allowing the connection to the endpoint to be made. If we decide to go with the original format it won't be a big deal; a bit more complex on the FE but we can iterate by Object key values still.